### PR TITLE
Provide module shards Config to Controller instead of tmp file

### PR DESCRIPTION
**Pull request by PeterSuna**
Provide custom moduleShardsConfig in controller instead of storing
moduleShardsConfig in tmp file: "/tmp/module-shards.conf".
This file was later read in initializing class ConfigurationImpl.

### CODE REVIEW

File: lighty-core/lighty-clustering/src/main/java/io/lighty/core/cluster/kubernetes/KubernetesClusteringHandlerImpl.java [hunk 1]
```diff
@@ -79,16 +74,10 @@ public void initClustering() {
...
-                List<String> memberRoles = akkaDeploymentConfig.getStringList("akka.cluster.roles");
-                String data = ClusteringConfigUtils.generateModuleShardsForMembers(memberRoles);
-                Files.write(Paths.get(ClusteringConfigUtils.MODULE_SHARDS_TMP_PATH),
-                        data.getBytes(StandardCharsets.UTF_8));
-                this.moduleShardsConfig = Optional.of(ClusteringConfigUtils.MODULE_SHARDS_TMP_PATH);
-                return;
-            } catch (IOException e) {
-                LOG.info("Tmp module-shards.conf file was not created - error received {}", e.getMessage());
-            }
+            List<String> memberRoles = akkaDeploymentConfig.getStringList("akka.cluster.roles");
```
Comment (ihrasko) [inline lighty-core/lighty-clustering/src/main/java/io/lighty/core/cluster/kubernetes/KubernetesClusteringHandlerImpl.java:77]: final

File: lighty-core/lighty-clustering/src/main/java/io/lighty/core/cluster/kubernetes/KubernetesClusteringHandlerImpl.java [hunk 2]
```diff
@@ -79,16 +74,10 @@ public void initClustering() {
...
+            String data = ClusteringConfigUtils.generateModuleShardsForMembers(memberRoles);
```
Comment (ihrasko) [inline lighty-core/lighty-clustering/src/main/java/io/lighty/core/cluster/kubernetes/KubernetesClusteringHandlerImpl.java:78]: final

**Review by ihrasko — CHANGES_REQUESTED** [review]
Move this PR to master branch.

**Comment by PeterSuna** [comment]
Duplicated with #952

**Pull request closed without merging**